src/drivers/intel/ssp/ssp.c

```diff
@@ -627,11 +627,11 @@ static int ssp_pre_start(struct dai *dai)
 
 	int ret = 0;
 
-	dai_info(dai, "ssp_pre_start()");
+	dai_info(dai, "ssp_port_start(), port_ref = %d", ssp->port_ref);
 
-	/* SSP active means bclk already configured. */
-	if (ssp->state[SOF_IPC_STREAM_PLAYBACK] == COMP_STATE_ACTIVE ||
-	    ssp->state[SOF_IPC_STREAM_CAPTURE] == COMP_STATE_ACTIVE)
+	ssp->port_ref++;
```

Comment:
Can we not use "state" here, I'm assuming this needs to be in a configured but inactive state ?

Reply:
the state checking is added back.